src/type-extensions.ts

```diff
@@ -56,6 +56,40 @@ type StarknetContractFactoryType = StarknetContractFactory;
 type StringMapType = StringMap;
 
 declare module "hardhat/types/runtime" {
+    interface L2Message {
```

Comment:
I think these interfaces could be along with the DevUtils class in a different file, since they're only used by DevUtils